### PR TITLE
Ocd 2885 (OCD-2820 - Resolve phantom logged in issue)

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

**Comment by kekey1** [comment]
Oh also please pull the latest development - pending release notes should not have anything else in them besides this ticket.

File: ocd-2885.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
+DROP TABLE IF EXISTS openchpl.ff4j_features;
```
Comment (andlar) [inline ocd-2885.sql]: Don't these need to be reversed so they can drop correctly?

**Comment by kekey1** [comment]
> Do we want to / can we / put these in a different schema than the regular CHPL data?

I had the same comment on the API ticket - so I think maybe we should!

File: dev/reset-test-only.sh [hunk 2]
```diff
@@ -10,6 +10,7 @@ if [ $# -ne 2 ]; then
     psql -Upostgres -f openchpl_preload.sql openchpl_test
     psql -Upostgres -f openchpl_api-key.sql openchpl_test
     psql -Upostgres -f openchpl_quartz.sql openchpl_test
+	psql -Upostgres -f openchpl_ff4j.sql openchpl_test
```
Comment (andlar) [inline dev/reset-test-only.sh]: It's tiny, but these various "different indentation" things irk the heck out of me

File: ocd-2885.sql [hunk 3]
```diff
@@ -0,0 +1,56 @@
+DROP SCHEMA IF EXISTS ff4j CASCADE;
+
+CREATE SCHEMA ff4j
```
Comment (andlar) [inline ocd-2885.sql]: Why not just do the `\i dev/ff4j-schema.sql` file thing? This looks like that same code, or at least SUPER-CLOSE

File: ocd-2885.sql [hunk 4]
```diff
@@ -0,0 +1,3 @@
+DROP SCHEMA IF EXISTS ff4j CASCADE;
+
+\i dev/ff4j-schema.sql
```
Comment (kekey1) [inline ocd-2885.sql]: Just tried running my v-next locally to get the schema change but the filename you have here is not the file that has the schema. Should be openchpl_ff4j.sql

**Pull request merged**